USER-INTERFACE-INTERFACE.md

```diff
@@ -386,6 +405,66 @@ database password. If you want to know whether the password you have is the corr
   try to connect to when it starts up next time.  It's a secret, so if you don't supply the `dbPasswordOpt` in the
   request you won't see it.
 
+* `paymentCurves`: These are parameters that define the Node's regard to quality of financial records created during interaction with
+  other Nodes. The goal is to determine whether the records have crossed boarders meaning that a certain action needs to be taken 
```

Comment:
maybe `parameters that define thresholds to determine when and how much to pay other nodes for routing and exit services and the expectations the node should have for receiving payments from other nodes for routing and exit services.`

Reply:
Thanks, this is helpful.